src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,149 @@
+import {
```

Comment:
This component also changed from Class -> Function, so the diff from the JS file is pretty significant.